plugins/modules/aci_interface_policy_spanning_tree.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: aci_interface_policy_spanning_tree
+short_description: Manage spanning tree interface policies (stp:IfPol)
+description:
+- Manage spanning tree interface policies on Cisco ACI fabrics.
+options:
+  stp_policy:
+    description:
+    - Name of the STP policy.
+    type: str
+    aliases: [ name ]
+  description:
+    description:
+    - The description for the policy.
+    type: str
+    aliases: [ descr ]
+  bpdu_guard:
+    description:
+    - BPDU-Guard state.
+    type: bool
+    default: false
+  bpdu_filter:
+    description:
+    - BPDU-Filter state.
+    type: bool
+    default: false
+  state:
+    description:
+    - Use C(present) or C(absent) for adding or removing.
+    - Use C(query) for listing an object or multiple objects.
+    type: str
+    choices: [ absent, present, query ]
+    default: present
+  name_alias:
+    description:
+    - The alias for the current object. This relates to the nameAlias field in ACI.
+    type: str
+extends_documentation_fragment:
+- cisco.aci.aci
+
+seealso:
+- name: APIC Management Information Model reference
+  description: More information about the internal APIC class B(stp:IfPol).
+  link: https://developer.cisco.com/docs/apic-mim-ref/
+author:
+- Eric Girard (@netgirard)
+'''
+
+# FIXME: Add more, better examples
+EXAMPLES = r'''
```

Comment:
Please add examples for query, query all and absent

Reply:
Done.